tests/network/test_node_storage.py

```diff
@@ -39,16 +31,8 @@ def test_one_node_stores_a_bunch_of_others(federated_ursulas, ursula_federated_t
 
     assert not newcomer.known_nodes
 
-    def start_lonely_learning_loop():
-        newcomer.start_learning_loop()
-        start = maya.now()
-        # Loop until the_chosen_seednode is in storage.
-        while the_chosen_seednode not in newcomer.node_storage.all(federated_only=True):
-            passed = maya.now() - start
-            if passed.seconds > 5:
-                pytest.fail("Didn't find the seed node.")
-
-    yield deferToThread(start_lonely_learning_loop)
+    newcomer.start_learning_loop(now=True)
 
-    # The known_nodes are all saved in storage (and no others have been saved)
-    assert list(newcomer.known_nodes) == list(newcomer.node_storage.all(True))
+    assert list(newcomer.known_nodes)
+    assert len(list(newcomer.known_nodes)) == len(list(newcomer.node_storage.all(True)))
```

Comment:
Heh - I know I wrote this but, what is `True` here? Perhaps kwargs :-)

Reply:
`True`.

Reply:
I see how it is.  You know there's a merge commit incoming so you include some garbage, complain about it later, and pin it on me.  `True` story. 

I thought we were friends.

Reply:
`False`

Reply:
Really, no need to make this change right now. :-)

Reply:
Shit man, I really left myself open to that one.  You win this round.